src/main/java/com/instrumentalapp/Connection.java

```diff
@@ -256,7 +256,7 @@ public void setShutdown(boolean shutdown) {
 				try {
 					worker.join(10000);
 				} catch (InterruptedException ie) {
-					LOG.severe("Failed to cleanly shutdown the Connection");
+					LOG.warning("Failed to cleanly shutdown the Connection");
```

Comment:
I agree this isn't probably severe, but is there an actual bad thing that will happen in this instance?

Reply:
At this point you are already exiting, so the bad thing is that you may miss some metrics that were enqueued but not yet sent.  This is the one I was least sure about.

Reply:
Yeah I was more concerned about dangling sockets or something.